.github/workflows/pr-test-e2e.yml

```diff
@@ -289,3 +289,22 @@ jobs:
           sleep 10
           curl http://localhost:8080/metrics
           kill %
+
+  # Consolidation job that ensures ALL tests pass before allowing merge
+  all-tests-complete:
+    name: All Tests Complete
```

Comment:
"all" is not wrong, if you have a narrow enough view, but maybe "both" is better?